.github/workflows/ci.yaml

```diff
@@ -0,0 +1,19 @@
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+name: ci
+jobs:
+  units:
+    run-on: ubuntu-latest
+    strategy:
+      matrix:
+        java: [8, 11]
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-java@v1
+      with:
+        java-version: ${{matrix.java}}
+    - run: java -version
```

Comment:
Not running real Maven build for now as Java 11 build would fail.

Reply:
You could take Java 11 out of the matrix.

Reply:
Nice idea. Removed 11.